app/assets/stylesheets/components/tiles/_article_tile.scss

```diff
@@ -189,3 +202,43 @@
   }
 }
 
+.article-tile-list {
+  list-style: none;
+  padding: 0;
+  margin-top: baseline-unit(9);
+}
+
+.article-tile-list__item {
+  @include font-size(30);
+  line-height: baseline-unit(6);
+  display: inline;
+  color: #000;
```

Comment:
We have a variable https://github.com/moneyadviceservice/publify/blob/popular_tags/app/assets/stylesheets/lib/_colors.scss#L4